apps/ml/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.9'
+services:
+  core_api:
+    build: .
+    container_name: "ml-api-container"
+    ports:
+      - "8000:80"
```

Comment:
I checked some of the ml-bridge code, please bind this to 8081 instead of 8000, this port is in sample .env for ml-bridge